README.md

```diff
@@ -52,6 +52,21 @@ The i8042 PS/2 controller emulates, at this point, only the
 [CPU reset command](https://wiki.osdev.org/%228042%22_PS/2_Controller#CPU_Reset)
 which is needed for announcing the VMM about the guest's shutdown.
 
+## Threat model
+
+For the serial console, there is no monitoring of the amount of data that is
+written to the `out` object.
+*Threat*: A malicious guest can fill up the host disk by generating a high
```

Comment:
For some reason the `Threat` is not on a new line when looking at the README. Can you add a newline here? Same for mitigation, limitations, and the last paragraph.